streaming/text/convert/enwiki_text.py

```diff
@@ -24,72 +24,73 @@ def parse_args() -> Namespace:
         '--in_root',
         type=str,
         required=True,
-        help='Directory path to store the input dataset',
+        help='Local directory path of the input raw dataset',
     )
     args.add_argument(
         '--out_root',
         type=str,
         required=True,
-        help='Directory path to store the output dataset',
+        help='Directory path to store the output MDS shard files',
     )
     args.add_argument(
         '--compression',
         type=str,
         default='zstd:7',
-        help='Compression algorithm to use. Default: zstd:7',
+        help='Compression algorithm to use. Defaults to ``zstd:7``',
```

Comment:
help strings

Reply:
Fixed in https://github.com/mosaicml/streaming/pull/171/commits/ad49770b6c7f35b3e143cdf0fc6f5f699f7da917.